sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/CertificatePolicy.java

```diff
@@ -107,7 +107,7 @@
      * Indicates if the same key pair will be used on certificate renewal.
      */
     @JsonProperty(value = "reuse_key")
-    private Boolean reuseKey;
+    private Boolean keyReusable;
```

Comment:
Same question here. Not a problem, but just want to make sure this change was intentional since other languages still use some form of `CertificateTransparency` and `ReusableKey`.

Reply:
Same comment reply here as above: this is very difficult from other languages, or was that intentional?

Reply:
same as above.